### ISSUE TITLE
Feature : setup log-opt with podman play kube

### ISSUE THREAD

**Issue by Archloy** [containers/podman#11727]
I'm tying to use kube.yaml directly with podman, I can set log-driver (I want to use journald) but I have to set also log-opt=tag="{{.ImageName}}" (because it's more clear, and driver journald refuse to work without that)
I can understand that setting log-path with kube is not possible but while we can't specify log-driver options directly into yaml file, I can't find any way to configure the log-driver. 
Or, this option is here only to set to "none" ? 
Thanks for help

**Comment by mheon** [comment]
Can you clarify exactly what you want to do - are you using `podman play kube` to make a pod, and want to set logging configuration for the created pod?

**Comment by Archloy** [comment]
@mheon Yes, exactly. While _podman play kube_ allow me to add option --log-driver, I also want to add --log-opt, as on podman run, so type something like : 
podman play kube --log-driver="journald" **--log-opt=<some opt>** kube.yaml 

And I can't do that directly on kube.yaml file (or I didn't see how) 
If there is no way to configure details on log-driver, the command line option "--log-driver" on podman play has an extremely restricted usage...
